lms/templates/journal.html

```diff
@@ -0,0 +1,39 @@
+<%def name="online_help_token()"><% return "course" %></%def>
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+from six import text_type
+%>
+<%page args="journal, journals_root_url" expression_filter="h"/>
+<article class="course" id="${journal.get('slug')}" role="region" aria-label="${journal.get('title')}">
+  <a href="${journals_root_url}/${journal.get('slug')}">
+    <header class="course-image">
+      <div class="cover-image">
+        <img src="${journal.get('card_image_url')}" alt="${journal.get('title')}" />
+        <div class="learn-more" aria-hidden="true">${_("LEARN MORE")}</div>
+      </div>
+    </header>
+    <div class="course-info" aria-hidden="true">
+      <h2 class="course-name">
+        <span class="course-organization">${journal.get('organization')}</span>
+        <span class="course-title">${journal.get('title')}</span>
+      </h2>
+      <%
+      if journal.get('access_length') is not None:
+          access_length_string = '{} months'.format(int(journal.get('access_length')/30))
```

Comment:
NIT:
From a technical standpoint, I don't know a better solution to this, but this gives me pause. Perhaps when we have more concrete products, we can add formatting options.

From a non-technical standpoint, we may want to talk to product since we could be saying it's 3 month access when it may be 90 days instead of 92. I imagine most will one year subscriptions, but just a thought.

Reply:
we could show the length in days, which would be the most accurate, but not sure the most user friendly